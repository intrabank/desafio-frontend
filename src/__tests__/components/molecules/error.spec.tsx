import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';
import Error from '@/components/molecules/error';

describe('Error', () => {
	it('should render error component', () => {
		const tree = renderer.create(<Error>Error</Error>).toJSON();
		expect(tree).toMatchSnapshot();
	});
});
