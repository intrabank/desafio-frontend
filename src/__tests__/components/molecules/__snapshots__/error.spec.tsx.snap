// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error should render error component 1`] = `
<span
  className="c-eFhfvJ c-eFhfvJ-bpRjqJ-color-red c-eFhfvJ-TkcIS-size-sm c-eFhfvJ-hZNfDR-weight-medium"
>
  Error
</span>
`;
